diff --git a/modules/fishnet/src/main/FishnetPlayer.scala b/modules/fishnet/src/main/FishnetPlayer.scala
index 4d463b4ab5..37b8d06f6a 100644
--- a/modules/fishnet/src/main/FishnetPlayer.scala
+++ b/modules/fishnet/src/main/FishnetPlayer.scala
@@ -28,11 +28,11 @@ final class FishnetPlayer(
         logger.info(e.getMessage)
       }
 
-  private val delayFactor = 0.011f
+  private val delayFactor = 0.003f
   private val defaultClock = Clock(Clock.LimitSeconds(300), Clock.IncrementSeconds(0))
 
   private def delayFor(g: Game): Option[FiniteDuration] =
-    if !g.bothPlayersHaveMoved then 2.seconds.some
+    if !g.bothPlayersHaveMoved then 1.seconds.some
     else
       for
         pov <- g.aiPov
@@ -41,7 +41,7 @@ final class FishnetPlayer(
         if totalTime > 20 * 100
         delay = (clock.remainingTime(pov.color).centis.atMost(totalTime)) * delayFactor
         accel = 1 - ((g.ply.value - 20).atLeast(0).atMost(100)) / 150f
-        sleep = (delay * accel).atMost(500)
+        sleep = (delay * accel).atMost(200)
         if sleep > 25
         millis = sleep * 10
         randomized = millis + millis * (ThreadLocalRandom.nextDouble() - 0.5)
